core/platform/models.py

```diff
@@ -27,8 +27,14 @@
 
 
 class _Platform(object):
+    """Parent Class for other classes to inherit
+
+    """
```

Comment:
Missing period(.) at the end. Also remove blank line at line-31.

Reply:
Done